### PR TITLE
Sh24 project changes

### PR DESCRIPTION
**Pull request by mybits**
PT: https://www.pivotaltracker.com/story/show/109625062

### CODE REVIEW

**Comment by tomsabin** [comment]
@mybits just to double check - is this meant to be merged to `staging` instead?

File: source/project-stories/sh24.erb [hunk 1]
```diff
@@ -1,37 +1,65 @@
...
+  number: "1,300"
+  text: "Tests orders"
+
+statistics_6:
+  number: "95%"
+  text: "People starting the order process through to completion"
+
+online_process:
+  title: "Taking the process from the doctor’s waiting room to online"
+  content: "Following a string of negative news headlines and a rise in sexual health STI rates in the Lambeth and Southwark areas, SH:24 came to us with an idea for a digital service: An online sexual health testing service for the residents of these areas. The vision was a self-service testing kit, ordered online, which could tackle long queues at inaccessible sexual health centers and dramatically increase public access to sexual health service."
```
Comment (tomsabin) [inline source/project-stories/sh24.erb]: This content is too large for the box unfortunately: 
screen shot 2016-01-04 at 11 20 18

Comment (mybits) [inline source/project-stories/sh24.erb]: Good catch, will ask Gram to make copy change.

**Comment by mybits** [comment]
Oh gee I did it again, sure, it's to be merge to `staging`, sorry!

File: source/project-stories/sh24.erb (hunk 1 above)
Comment (mybits) [inline source/project-stories/sh24.erb]: That's fixed with copy changes, will open new PR :)

**Pull request closed without merging**